src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,6 +1,15 @@
 package frc.robot;
 
 public class RobotMap {
-
+    //                          ARM
+    // ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
+    public final static int ARM_MOTOR_PORT = 8;
+    public final static int ARM_ENCODER_PORT = 0;
+    public final static double ARM_GEAR_RATIO = 20 / 1;
+    public final static int ARM_ENCODER_CPR = 42;
```

Comment:
```suggestion
    public final static int ARM_NEO_ENCODER_CPR = 42;
```
Since there are two encoders, you should probably be more specific about which